packaging/os/yaourt.py

```diff
@@ -0,0 +1,385 @@
+---
+module: yaourt
+short_description: Manage packages with I(yaourt)
+description:
+    - Manage packages with the I(yaourt) package manager, an wrapper around pacman, 
+      which is used on Arch Linux and its variants to access the AUR.
+version_added: "1.0"
+author: Afterburn
+notes: []
+requirements: []
+options:
+    name:
+        description:
+            - Name of the package to install, upgrade, or remove.
+        required: false
+        default: null
+
+    state:
+        description:
+            - Desired state of the package.
+        required: false
+        default: "present"
+        choices: ["present", "absent", "latest"]
+
+    recurse:
+        description:
+            - When removing a package, also remove its dependencies, provided
+              that they are not required by other packages and were not
+              explicitly installed by a user.
+        required: false
+        default: "no"
+        choices: ["yes", "no"]
+        version_added: "1.3"
+
+    force:
+        description:
+            - Force remove package, without any checks.
+        required: false
+        default: "no"
+        choices: ["yes", "no"]
+        version_added: "2.0"
+
+    update_cache:
+        description:
+            - Whether or not to refresh the master package lists. This can be
+              run as part of a package installation or as a separate step.
+        required: false
+        default: "no"
+        choices: ["yes", "no"]
+
+    upgrade:
+        description:
+            - Whether or not to upgrade whole system
+        required: false
+        default: "no"
+        choices: ["yes", "no"]
+        version_added: "2.0"
+        
+    upgrade_aur:
+        description:
+            - Whether or not to upgrade whole system and aur-packages 
+        required: false
+        default: "no"
+        choices: ["yes", "no"]
+        version_added: "2.0"
+        
+    check_kernelupdate:
+        description:
+            - Checks if the kernel has been updated
+        default: "no"
+        choices: ["yes", "no"]
+        version_added: "2.0"
+'''
+
+EXAMPLES = '''
+# Install package foo
+- yaourt: name=foo state=present
+
+# Upgrade package foo
+- yaourt: name=foo state=latest update_cache=yes
+
+# Remove packages foo and bar
+- yaourt: name=foo,bar state=absent
+
+# Recursively remove package baz
+- yaourt: name=baz state=absent recurse=yes
+
+# Run the equivalent of "yaourt -Sy" as a separate step
+- yaourt: update_cache=yes
+
+# Run the equivalent of "yaourt -Su" as a separate step
+- yaourt: upgrade=yes
+
+# Run the equivalent of "yaourt -Sua" as a separate step
+- yaourt: upgrade_aur=yes
+
+# Diff "yaourt -Qi linux(-lts)" to "uname -r" as a separate step
+- yaourt: check_kernelupdate=yes
+
+# Run the equivalent of "yaourt -Rdd", force remove package baz
+- yaourt: name=baz state=absent force=yes
+'''
+
+import json
+import shlex
+import os
+import re
+import sys
+
+YAOURT_PATH = "/usr/bin/yaourt"
```

Comment:
use module function that finds bin path ... just in case someone decides to move it